zerver/openapi/zulip.yaml

```diff
@@ -12014,6 +12014,147 @@ paths:
+                    invitation will never expire. If unspecified, the server will use a default
+                    value (`INVITATION_LINK_VALIDITY_MINUTES`) for when the invitation will expire.
+
+                    **Changes**: New in Zulip 6.0 (feature level 126). Previously, there was an
+                    `invite_expires_in_days` parameter, which specified the duration in days instead
+                    of minutes.
+                  type: integer
+                  nullable: true
+                  example: 14400
+                invite_as:
+                  description: |
+                    The [organization-level role](/api/roles-and-permissions) of the user that is
+                    created when the invitation is accepted.
+
+                    Users can only create invitation links for
+                    [roles with equal or stricter restrictions](/api/roles-and-permissions#permission-levels)
+                    as their own. For example, a moderator cannot invite someone to be an owner
+                    or administrator, but they can invite them to be a moderator or member.
+
+                    **Changes**: In Zulip 4.0 (feature level 61), added support for inviting
+                    users as moderators.
+                  type: integer
+                  enum:
+                    - 100
+                    - 200
+                    - 300
+                    - 400
+                    - 600
+                  default: 400
+                  example: 600
+                stream_ids:
+                  description: |
+                    A list containing the [IDs of the streams](/api/get-stream-id) that the
+                    newly created user will be automatically subscribed to if the invitation
+                    is accepted. If unspecified, it will be set to empty list. If the list is
+                    empty, then the new user will not be subscribed to any streams.
+                  type: array
+                  items:
+                    type: integer
+                  default: []
+                  example: [1, 2]
+            encoding:
+              invite_expires_in_minutes:
+                contentType: application/json
+              stream_ids:
+                contentType: application/json
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      invite_link:
+                        type: string
+                        description: |
+                          The URL of the [reusable invitation link](/help/invite-new-users#create-a-reusable-invitation-link).
```

Comment:
Say a few more words, like adding "that was created by this request." at the end of this sentence.